src/Controller/Admin/HomeController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use Core\AdminControllerAbstract;
+
+class HomeController extends AdminControllerAbstract
+{
+    public function indexAction(): AdminControllerAbstract
```

Comment:
euh ... nan, ca retourne pas $this, mais $this->renderView();

Reply:
Je sais. AdminControllerAbstract hérite de DefaultControllerAbstract.
D'ailleurs j'ai essayé avec DefauktControllerAbstract et PHPStan me renvoit une erreur